core/src/main/scala/org/apache/spark/deploy/history/HistoryAppStatusStore.scala

```diff
@@ -45,20 +45,20 @@ private[spark] class HistoryAppStatusStore(
   override def executorList(activeOnly: Boolean): Seq[v1.ExecutorSummary] = {
     val execList = super.executorList(activeOnly)
     logUrlPattern match {
-      case Some(pattern) => execList.map(replaceLogUrls(_, pattern))
+      case Some(_) => execList.map(replaceLogUrls)
       case None => execList
     }
   }
 
   override def executorSummary(executorId: String): v1.ExecutorSummary = {
     val execSummary = super.executorSummary(executorId)
     logUrlPattern match {
-      case Some(pattern) => replaceLogUrls(execSummary, pattern)
+      case Some(_) => replaceLogUrls(execSummary)
       case None => execSummary
     }
   }
 
-  private def replaceLogUrls(exec: v1.ExecutorSummary, urlPattern: String): v1.ExecutorSummary = {
```

Comment:
[SPARK-26311](https://issues.apache.org/jira/browse/SPARK-26311) defines this method with `urlPattern` and after SPARK-26792, change to use `logUrlPattern` held by `ExecutorLogUrlHandler`